web/src/features/chat/components/ChatMessage.tsx

```
@@ -1,96 +1,421 @@
+
+    const showFiles = (isBuildingPhase || isCompletedPhase) && totalFiles > 0
+
+    const showSummary =
+        Boolean(summary) ||
+        (isCompletedPhase && content && content !== plan && content !== thoughts)
+    const summaryText = summary || (isCompletedPhase ? content : '')
+
+    const [displayedPlan, setDisplayedPlan] = React.useState('')
+    const [displayedSummary, setDisplayedSummary] = React.useState('')
+
+    React.useEffect(() => {
```

ESLint

[error] React Hook "React.useEffect" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)